mappings/net/minecraft/entity/player/PlayerEntity.mapping

```diff
@@ -110,6 +110,8 @@ CLASS arz net/minecraft/entity/player/PlayerEntity
 	METHOD dx getScore ()I
 	METHOD dy vanishCursedItems ()V
 	METHOD f attack (Lahm;)V
+	METHOD f tryAddStackToInv (Laxx;)Z
```

Comment:
`giveItemStack`?

Reply:
I chose `tryAddStackToInv` because if it fails, the stack is not dropped as an entity or anything. `giveItemStack` implies that it would because the `/give` command handles dropping as an item.

Reply:
I feel the return value is enough to indicate the success/failure without the need to describe this intent in the method name.